src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -18,7 +18,7 @@ public Intake() {
     rollerMotor = new TalonFX(mapIntake.ROLLER_CAN, "rio");
   }
 
-  public void setRollerOutput(double speed) {
+  public void setIntakeRollerSpeed(double speed) {
```

Comment:
If we're going to store the intake speeds as measures, we should have the intake methods take measures as their parameters (not doubles)